lib/impure/db_postgres.nim

```diff
@@ -168,8 +169,8 @@ proc newRow(L: int): Row =
 proc setupQuery(db: DbConn, query: SqlQuery,
                 args: varargs[string]): PPGresult =
   # s is a dummy unique id str for each setupQuery query
-  let s = "setupQuery_Query_" & string(query)
-  var res = pqprepare(db, s, dbFormat(query, args), 0, nil)
+  let s = "Q_" & $random(100000) & "_" & string(query)
```

Comment:
Maybe generate an `Oid` to ensure randomness: http://nim-lang.org/docs/oids.html